openstack/nova/templates/console-deployment.yaml

```
@@ -0,0 +1,7 @@
+{{ $envAll := . }}
+{{- range $name, $config := .Values.consoles }}
+  {{- if $config.enabled }}
+---
+{{ tuple $envAll $name $config | include "nova.console_deployment" }}
```

Would you consider this for legibility?
```suggestion
    {{- tuple $envAll $name $config | include "nova.console_deployment" }}
```

---

ah, sorry that doesn't work. nevermind.